ownership-chain/runtime/Cargo.toml

```diff
@@ -90,6 +90,8 @@ pallet-evm-precompile-simple = { workspace = true }
 
 [dev-dependencies]
 static_assertions = { workspace = true }
+precompile-utils = { workspace = true, features = ["testing"] }
+evm = { workspace = true }
```

Comment:
what about include also `evm` dependency when testing as `precompile-utils` ?

Reply:
disregard, conceptual misunderstanding by my side

Reply:
`evm` and `precompile-utils` creates are already included only in tests (aka `[dev-dependecies]).